backend/src/main/java/com/google/sps/data/Participant.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+/** A user who want to be matched. */
+public final class Participant {
+
+  private final long id;
+  private final String ldap; // Google username
+  private final long timeAvailableUntil; // Time user is available until
+  private final String timezone; // User timezone
+  private final int duration; // How long user wants to chat
+  private final long timestamp; // Time of submitted form
```

Comment:
Each of these comments could be the javadocs for the field. So instead of 

```
  private final int duration; // How long user wants to chat
  private final long timestamp; // Time of submitted form
```

It could be
```
/** How long the user wants to chat. */
private final int duration;
/** Time of submitted form. */
private final long timestamp;
```

This is the standard way of documenting Java fields and allows tools to auto-generate docs pages for your classes. It also helps the IDE show hints when fields are highlighted with the cursor or referenced elsewhere.